app/controllers/api/v1/discussions_controller.rb

```diff
@@ -4,7 +4,11 @@ class Api::V1::DiscussionsController < Api::V1::ApiController
   before_action :find_exchange, only: [:show]
 
   def index
-    @exchanges = current_resource_owner.unhidden_discussions.viewable_by(current_resource_owner).page(params[:page]).for_view
+    @exchanges = current_resource_owner
+      .unhidden_discussions
```

Comment:
Place the . on the previous line, together with the method call receiver.<br>Align the operands of an expression in an assignment spanning multiple lines.